README.rst

```diff
@@ -45,11 +56,39 @@ upon by some of the world's leading scientists and engineers. If you need to
 manipulate numbers on a computer and display or publish the results, give
 SciPy a try!
 
-For the installation instructions, see INSTALL.rst.txt_.
+For the installation instructions, see `our install
+guide <https://scipy.github.io/devdocs/getting_started.html#installation>`__.
+
+
+Call for Contributions
+----------------------
+
+The SciPy project welcomes your expertise and enthusiasm!
+
+Small improvements or fixes are always appreciated; issues labeled as "good
+first issue" may be a good starting point. Have a look at `our contributing
+guide <http://scipy.github.io/devdocs/dev/hacking.html>`__.
+
+Writing code isn’t the only way to contribute to SciPy. You can also:
+
+- review pull requests
+- triage issues
+- develop tutorials, presentations, and other educational materials
+- maintain and improve `our website <https://github.com/scipy/scipy.org>`__
+- develop graphic design for our brand assets and promotional materials
+- translate website content
+- help with outreach and onboard new contributors
+- write grant proposals and help with other fundraising efforts
+
+If you’re unsure where to start or how your skills fit in, reach out! You can
+ask on the mailing list or here, on GitHub, by opening a new issue or leaving a
+comment on a relevant issue that is already open.
 
-We appreciate and welcome contributions. If you would like to take part in
-SciPy development, take a look at the file CONTRIBUTING.rst_.
+Our preferred channels of communication are all public.
```

Comment:
What do you mean by this?

Reply:
That there is no back-channel and everything we do is public and you can audit it. But again this is from NumPy.

Reply:
Exactly what a back-channel owner would say 😏

Reply:
I removed but for reference this is the inspiration

> Our preferred channels of communication are all public, but if you’d like to speak to us in private first, contact our community coordinators at numpy-team@googlegroups.com or on Slack (write numpy-team@googlegroups.com for an invitation).
>We also have a biweekly community call, details of which are announced on the mailing list. You are very welcome to join.

And we probably should have something like this. But that's another discussion.